.github/workflows/flathub-data-checker.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Check if external data changed'
```

Comment:
You were supposed to push the change into `new-pr` branch which doesn't contain the `.github` folder. Do you mind removing it?

Reply:
@bilelmoussaoui folder .github removed. Thanks